projects/components/src/lib/result-table/result-table.component.ts

```diff
@@ -10,14 +11,13 @@ import { ResultTableModel } from './result-table.model';
 export class ResultTableComponent {
     @Input() public loading: boolean = false;
     @Input() public error?: Error;
-
-    @Input() model: ResultTableModel<any> | undefined | null; // TODO need to deal with the flaw of angular's async pipe
-
+    @Input() public model: ResultTableModel<any> | undefined | null; // TODO need to deal with the flaw of angular's async pipe
     @Input() public header!: TemplateRef<any>;
     @Input() public body!: TemplateRef<any>;
     @Input() public pagination: TemplateRef<any> | undefined | null;
-
+    @Input() public options!: ResultTableOptions | undefined | null;
```

Comment:
We can default this to `{ hidePagination: false }`.

Reply:
done